platform_plugin_ontask/api/utils.py

```diff
@@ -18,7 +19,21 @@
     WorkflowIDNotSetError,
 )
 
-DEFAULT_DATA_SUMMARY_CLASS = CompletionDataSummary
+log = logging.getLogger(__name__)
+
+
+def ontask_log_from_response(response: Response) -> str:
+    """
+    Return a log message from a response object.
+
+    Arguments:
+        response (Response): The response object.
+    """
+    return (
+        f"{response.request.method} {response.url} | "
+        f"status-code={response.status_code} | "
+        f"response={response.text}"
+    )
```

Comment:
Django already logs some of this information. Should we log it again? I think it's more valuable to log the response text only. Also, logging directly from the view removes a level of abstraction from the implementation.

Reply:
I agree.
Done: https://github.com/eduNEXT/platform-plugin-ontask/commit/8dc63f97ba7b6002a943897a735df77750d8ba68